html/login.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>로그인</title>
+    <link rel="icon" type="image/svg" href="/image/logo.svg" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    />
+    <link rel="stylesheet" href="/css/auth.css" />
+  </head>
+
+  <body class="body_login">
+    <header>
+      <a href="/">
+        <img class="logo" src="/image/logo.svg" alt="판다마켓 로고" />
+      </a>
+      <a class="logo-title" href="/">
+        <img src="/image/logo_title.svg" alt="판다마켓 로고 글씨" />
+      </a>
+    </header>
+
+    <main>
+      <form>
+        <div class="input-box">
+          <label for="email">이메일</label>
+          <input name="email" id="email" type="email" />
+        </div>
+
+        <div class="input-box">
+          <label for="password">비밀번호</label>
+          <div class="pw-content">
+            <input name="password" id="password" type="password" />
+            <img
+              class="eye_icon"
+              src="/image/password_toggle_icon.svg"
+              alt="비밀번호를 보이지 않게 하는 눈 아이콘"
+            />
```

Comment:
❗️ 수정요청  
해당 요소는 클릭 시 비밀번호가 보이고 안 보이게 만드는 요소가 될 것이므로 button요소로 작성하시는 것을 추천드려요.

```suggestion
            <button type="button">
              <img
                class="eye_icon"
                src="/image/password_toggle_icon.svg"
                alt="비밀번호를 보이지 않게 하는 눈 아이콘"
              />
            </button>
```